### PR TITLE
Content fixes

### PR DESCRIPTION
**Pull request by bwsw**

### CODE REVIEW

File: content/developers.html [hunk 1]
```diff
@@ -146,6 +146,27 @@ <h1 id="indicators">Developer Resources for Apache CloudStack</h1>
 
 <div class="col-lg-8">
 
+<h2>CloudStack Repositories</h2>
+<p><a href="https://github.com/apache/cloudstack" target="_blank">System Source Code</a>. The main CloudStack-related project,
```
Comment (DaanHoogland) [inline content/developers.html]: these paths are worth mentioning but not the "origin of truth". gitbox.apache.org contains the repository and is sync'd to github!

File: content/developers.html [hunk 2]
```diff
@@ -256,30 +278,25 @@ <h3>Further Reading</h3>
...
                 
-<h3 class="panel-title">Git Repositories</h3>
+<h3 class="panel-title">CloudStack Git Repositories</h3>
               
 </div>
               
 <div class="panel-body">
                 
-<p>The git repositories are hosted on Apache infrastructure, and can be found here:</p>
+<p>The git repositories are hosted on GitHub infrastructure, and can be found here:</p>
```
Comment (DaanHoogland) [inline content/developers.html]: no, this is not where it is hosted. This is where it is synced to. it is hosted at gitbox.apache.org

File: content/developers.html [hunk 3]
```diff
@@ -256,30 +278,25 @@ <h3>Further Reading</h3>
...
+<li><a href="https://github.com/apache/cloudstack-cloudmonkey" target="_blank">Apache CloudStack Cloudmonkey source code</a></li>
+<li><a href="https://github.com/apache/cloudstack-documentation" target="_blank">Documentation</a></li>
 <li><a href="https://github.com/apache/cloudstack-www" target="_blank">Apache CloudStack Website</a></li>
 </ul>
 
 <p>To get the most recent source for Apache CloudStack, use:</p>
 
 <pre>
-git clone https://gitbox.apache.org/repos/asf/cloudstack.git
+git clone https://github.com/apache/cloudstack.git
```
Comment (DaanHoogland) [inline content/developers.html]: please do not merge this

File: content/developers.html (hunk 1 above)
Comment (bwsw) [reply content/developers.html]: @DaanHoogland removed the part.

File: content/developers.html (hunk 2 above)
Comment (bwsw) [reply content/developers.html]: @DaanHoogland returned back.

File: content/developers.html (hunk 3 above)
Comment (DaanHoogland) [reply content/developers.html]: I might have over reacted in this instance. In this above piece I'd put both paths but keep in mind that apache hosts the code for legal reasons. Suggestion:
<pre>
git clone https://gitbox.apache.org/repos/asf/cloudstack.git
</pre>
<p>or the mirror at</p>
<pre>
git clone https://github.com/apache/cloudstack.git
</pre>

**Comment by bwsw** [comment]
@DaanHoogland I close this PR and open new to master, because Paul and Rohit merged asf-site to master as it should be.

**Pull request closed without merging**